### PR TITLE
Exploration hardsuit's can now locate research disk, objectives and the exploration shuttle

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

The exploration hardsuit will now point towards research disks, objective items and the exploration shuttles. (See testing photographs)

## Why It's Good For The Game

Quality of life upgrade, no longer requiring to have a GPS device open on a second monitor (which is inconvenient if you only have 1 monitor). Also allows for easy locating of research disks, meaning explorers are much more likely to explore the ruins and collect the disks.

## Testing Photographs and Procedure

image

image

image

## Changelog
:cl:
add: Exploration hardsuits now have a HUD which points them towards research disks, objectives and the exploration shuttle.
/:cl:

### CODE REVIEW

**Comment by Isy232** [comment]
As someone with 2 monitors, I can't imagine trying to use a GPS while clearing a ruin with only one.

Well, could cover up most of the chat box, but that would hamper communications with other explorers.

Good stuff.

File: code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/nuke_ruin.dm [hunk 1]
```diff
@@ -41,6 +41,7 @@
 
 /obj/item/disk/nuclear/decommission/ComponentInitialize()
 	AddComponent(/datum/component/gps, "AUTH0", TRUE)
+	AddComponent(/datum/component/tracking_beacon, "expl", null, null, TRUE, "#f3d594", TRUE, TRUE)
```
Comment (itsmeow) [inline code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/nuke_ruin.dm:44]: `expl` being a magic string is 👎🏻

File: code/datums/components/team_monitor.dm [hunk 2]
```diff
@@ -49,7 +49,7 @@ GLOBAL_LIST_EMPTY(tracker_beacons)
 
 /proc/get_all_beacons_on_frequency(frequency, base_frequency)
 	if(!frequency)
-		return list()
+		return GLOB.tracker_beacons["[base_frequency]-GLOB"]
```
Comment (itsmeow) [inline code/datums/components/team_monitor.dm:52]: Why this change?

**Review by itsmeow — CHANGES_REQUESTED** [review]

File: code/datums/components/team_monitor.dm (hunk 2 above)
Comment (PowerfulBacon) [reply code/datums/components/team_monitor.dm:52]: If no frequency was provided then global signals wouldn't be displayed on the monitor HUD

**Comment by PowerfulBacon** [comment]
Research disks should disable their beacon when picked up

**Comment by PowerfulBacon** [comment]
Ships land before generation finishes

**Comment by PowerfulBacon** [comment]
Shuttles dont blow up when destroyed and shuttle health is wacky
